test/brain/redis_store_test.exs

```diff
@@ -0,0 +1,105 @@
+defmodule Elix.RedisStore.RedisStoreTest do
+  use ExUnit.Case, async: true
+  alias Elix.Brain.RedisStore
+
+  setup do
+    RedisStore.start_link
+    Redix.command!(:redis, ["FLUSHDB"])
```

Comment:
Ended up adding a [`RedisStore.delete_all`](https://github.com/stevegrossi/hedwig_brain/blob/master/lib/brain/redis_store.ex#L52) function. I considered adding something similar to the `ProcessStore` but figure I can wait until there's actually a use for it. I also considered giving it a clever name like `Brain.amnesia!` but when I get clever I tend to regret it.